Document workaround for Kubernetes IPv6 Clusters using KubernetesClient (#3343)
fabric8io/kubernetes-client: doc/Faq.md

# Frequently Asked Questions:

> ### How to I use KubernetesClient with IPv6 Kubernetes Clusters?

Reviewer: ```suggestion
### How do I use KubernetesClient with IPv6 Kubernetes Clusters?
```

> We're aware of this [issue](https://github.com/fabric8io/kubernetes-client/issues/2632) in Fabric8 Kubernetes Client. Unfortunately, this is caused by OkHttp which we can't upgrade. You can check suggested workaround here:

Reviewer: Your statement sounds like there's nothing that can be done and we're suggesting just a palliative workaround. However OkHttp claims that this is OK and that the end-user is responsible to switch to OkHttp 4 if interested.
There's a similar discussion around [retrofit](https://github.com/square/retrofit/issues/3566)

```suggestion
We're aware of this [issue](https://github.com/fabric8io/kubernetes-client/issues/2632) in Fabric8 Kubernetes Client. Unfortunately, this is caused by the OkHttp transitive dependency. You can check suggested workaround here:
```
